src/platforms/javascript/common/troubleshooting/index.mdx

```diff
@@ -103,7 +103,99 @@ Sentry.init({
 });
 ```
 
-Once configured, all events will be sent to the `/tunnel` endpoint. This solution, however, requires an additional configuration on the server, as the events now need to be parsed and redirected to Sentry. Here's an example for your server component:
+Once configured, all events will be sent to the `/tunnel` endpoint. This solution, however, requires an additional configuration on the server, as the events now need to be parsed and redirected to Sentry.
+
```

Comment:
```suggestion
Once configured, all events will be sent to the `/tunnel` endpoint. This will require an additional configuration on the server because the events will need to be parsed and redirected to Sentry.

```